### ISSUE TITLE
Consider removing `text-align: justify;`

### ISSUE THREAD

**Issue by ntwb** [WordPress/twentytwenty#629]
Considering removing `text-align: justify;` as there are accessibility issues when used:

Detected via stylelint-a11y plugin [no-text-align-justify](https://github.com/YozhikM/stylelint-a11y/blob/master/src/rules/no-text-align-justify/README.md) rule

https://www.w3.org/TR/WCAG20-TECHS/G169.html#G169-description: 
> Many people with cognitive disabilities have a great deal of trouble with blocks of text that are justified (aligned to both the left and the right margins). The spaces between words create "rivers of white" running down the page, which can make the text difficult for some people to read. This failure describes situations where this confusing text layout occurs. The best way to avoid this problem is not to create text layout that is fully justified.

https://github.com/WordPress/twentytwenty/blob/master/style.css#L2797-L2799

I've no strong opinion on removing this as I'm not sure where its actually used ;)

**Comment by grahamarmfield** [comment]
I support this change as fully justified text in websites does cause issues for some people - inc some people with dyslexia (approx 10% of population).

**Comment by nielslange** [comment]
@ntwb The class `.has-text-align-justify` which contains `text-align: justify;` isn't used anywhere on te site and with https://github.com/WordPress/twentytwenty/pull/638 I removed it.
